src/frontend/apps/drive/src/features/i18n/translations.json

```diff
@@ -308,7 +311,9 @@
           "last_update": "Dernière modification",
           "empty": {
             "caption": "Déposez vos fichiers ici",
-            "cta": "Ou cliquez sur \"Créer\""
+            "caption_no_create": "Vous n'avez pas les droits pour ajouter des fichiers",
```

Comment:
WDYT: "Aucun documents ici" ?

Reply:
As we already inform in the CTA that readers cannot add files, at least we need to tell the user that there is no files